frontend/src/components/core/Sidebar/SidebarNav.tsx

```diff
@@ -135,16 +133,19 @@ const SidebarNav = ({
                       }
                     }}
                   >
-                    {pageIcon && pageIcon.length ? (
+                    {pageIcon && pageIcon.length && (
                       <EmojiIcon size="lg">{pageIcon}</EmojiIcon>
-                    ) : (
-                      <Icon
-                        color="darkenedBgMix100"
-                        content={Description}
-                        size="lg"
-                      />
                     )}
-                    <span>{pageName.replace(/_/g, " ")}</span>
+                    <StyledSidebarLinkText
+                      isActive={pageScriptHash === currentPageScriptHash}
+                    >
+                      <OverflowTooltip
+                        content={pageName.replace(/_/g, " ")}
+                        placement={Placement.AUTO}
+                      >
+                        {pageName.replace(/_/g, " ")}
```

Comment:
nit: would recommend setting `pageName.replace(/_/g, " ")` to some intermediary const above so that the call to `.replace()` isn't duplicated

Reply:
Makes sense, updated in https://github.com/streamlit/streamlit/pull/5445/commits/fa74dd75594247e2f49675c29b6fcc3e7c9be126!